src/PostList/PostList.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FC } from 'react';
+import { CommentList } from '../CommentList/CommentList';
+
+interface Props {
+  listOfPosts: CompletedPost[];
```

Comment:
странное сочетание интерфейса и названия пропсов, на мой взгляд
```suggestion
  posts: PostWithUser[];
```